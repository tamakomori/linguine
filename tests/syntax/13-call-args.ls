func main() {
    callee(123, 1.23, "abc");
}

func callee(a, b, c) {
    print("a=" + a);
    print("b=" + b);
    print("c=" + c);
}
