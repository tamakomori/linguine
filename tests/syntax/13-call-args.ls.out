a=123
b=1.230000
c=abc
